src/eslint-plugin-adeira/README.md

```diff
@@ -1,20 +1,20 @@
-Rules used at Kiwi.com in the Incubator tribe. You most probably do not need to use this package directly - use [@kiwicom/eslint-config](https://www.npmjs.com/package/@kiwicom/eslint-config) instead. This package exists basically only to workaround some Eslint limitations.
+Rules in Adeira projects. You most probably do not need to use this package directly - use [@adeira/eslint-config](https://www.npmjs.com/package/@adeira/eslint-config) instead. This package exists basically only to workaround some Eslint limitations.
 
 # Installation
 
 You'll first need to install [ESLint](http://eslint.org). Next, install this plugin:
 
 ```
-yarn add --dev eslint-plugin-kiwicom-incubator
+yarn add --dev eslint-plugin-adeira
 ```
 
 # Usage
 
-Add `kiwicom-incubator` to the plugins section of your `.eslintrc` configuration file:
+Add `adeira` to the plugins section of your `.eslintrc` configuration file:
 
 ```json
 {
-  "plugins": ["eslint-plugin-kiwicom-incubator"]
+  "plugins": ["eslint-plugin-eslint-plugin-adeira"]
```

Comment:
This seems to be wrong (duplicated `eslint-plugin-`).

Reply:
Probably from search and replace, yes, I will fix it.